pkgs/applications/misc/hunter/default.nix

```diff
@@ -0,0 +1,51 @@
+{ lib
+, stdenv
+, pkg-config
+, rustPlatform
+, fetchFromGitHub
+, makeWrapper
+, glib
+, gst_all_1
+, CoreServices
+, IOKit
+, Security
+}:
+
+rustPlatform.buildRustPackage rec {
+  pname = "hunter";
+  version = "1.3.5";
+
+  src = fetchFromGitHub {
+    owner = "06kellyjac";
+    repo = "hunter";
+    rev = "2484f0db580bed1972fd5000e1e949a4082d2f01";
```

Comment:
```suggestion
    rev = "v${version }";
```

Reply:
Reverting the commit to the original version means I have to add a rust overlay. The program only worked with nightly rust before.

Do you still want me to do that?

Reply:
Please fetch the release and the two commits as patches instead of fetching the fork branch.

Reply:
The commits ((this)[https://github.com/06kellyjac/hunter/commit/a5943578e1ee679c8bc51b0e686c6dddcf74da2a] and (this)[https://github.com/06kellyjac/hunter/commit/1b8de9248312878358afaf1dac569ebbccc4321a]) don't get patched as expected *on top* of the 1.3.5 version. They're based on the latest commit, which is more than 70 commits ahead of 1.3.5. I guess I should switch the version to the latest commit as well then, and change the version to `2020-05-25-unstable`.
What do you think @SuperSandro2000 ?

Reply:
Sure. If you think the software works better at this commit and does not contain any new bugs.

Reply:
I found something interesing: even if `owner = "rabite0"`, specifying the latest commit from 06kellyjac's fork results in fetching that version of the repo and it builds just fine.
Should I still have the extra commits as patches or is it fine to let the `rev` be the latest commit from that fork?

Reply:
* ``020-05-25-unstable``

> I found something interesing: even if owner = "rabite0", specifying the latest commit from 06kellyjac's fork results in fetching that version of the repo and it builds just fine.

GitHub thought this is a good "feature" but it is just a security nightmare. Please fetch the commit from the repo where it belongs/originates to.